src/js/components/auditlogs/auditlogs.js

```diff
@@ -189,49 +197,34 @@ export const AuditLogs = ({ events, getAuditLogsCsvLink, getAuditLogs, getUserLi
               InputProps={{ ...params.InputProps }}
             />
           )}
-          renderOption={option => option.email || option}
           style={{ maxWidth: 250 }}
         />
         <Autocomplete
+          {...autoSelectProps}
           id="audit-log-type-selection"
           key={`audit-log-type-selection-${filterReset}`}
-          autoSelect
-          filterSelectedOptions
-          getOptionLabel={option => option.title || option}
-          handleHomeEndKeys
           inputValue={type}
           onInputChange={onTypeFilterChange}
           options={AUDIT_LOGS_TYPES}
           renderInput={params => (
             <TextField {...params} label="Filter by change" placeholder="Type" InputLabelProps={{ shrink: true }} InputProps={{ ...params.InputProps }} />
           )}
-          renderOption={option => option.title}
           style={{ marginLeft: 7.5 }}
         />
         <Autocomplete
+          {...autoSelectProps}
           id="audit-log-type-details-selection"
-          autoSelect
           disabled={!type}
-          filterSelectedOptions
-          getOptionLabel={option => option.email || option}
-          handleHomeEndKeys
           inputValue={detail}
           onInputChange={onDetailFilterChange}
           options={detailOptions}
           renderInput={params => <TextField {...params} placeholder={detailsMap[type] || '-'} InputProps={{ ...params.InputProps }} />}
-          renderOption={option => option.email || option}
           style={{ marginRight: 15, marginTop: 16 }}
         />
         <div />
         <TimerangePicker endDate={endDate} onChange={onTimeFilterChange} startDate={startDate} />
-        <div style={{ gridColumnStart: 2, gridColumnEnd: 4 }}>
-          <TimeframePicker
-            classNames="margin-left margin-right inline-block"
-            onChange={onTimeFilterChange}
-            endDate={endDate}
-            startDate={startDate}
-            tonight={tonight}
-          />
+        <div style={{ gridColumnStart: 2, gridColumnEnd: 4, marginLeft: -7.5 }}>
```

Comment:
basically whenever a file starts with `import React from 'react'` any html looking components will be evaluated by react and the passed arguments will be handled by react... afaik most of the default html entities will just be translated to react components this way...
and if you pass an argument that the component in questions doesn't understand, this will be written to the html as is (and results in errors like we have on the login page 🙈)

Reply:
haha, oh my :see_no_evil: 

But is this not considered poor practice?

Reply:
the behaviour that results in errors? yep... but there is [a task](https://tracker.mender.io/browse/MEN-3530) for it 😬...